### DISCUSSION TITLE
Library Extension Support

### DISCUSSION THREAD

**Discussion by eirenicon** [bookwyrm-social/bookwyrm#2877] [Ideas]
Support/ integration with the Library Extension browser plugin (FF, Chrome) would be an outstanding adjunct to Bookwyrm.   See: https://www.libraryextension.com/

**Comment by mouse-reeve** [comment]
That does sound really cool! From a brief perusal of their website, it looks like they would need to support BookWyrm by adding to their list of approved websites. If they were down to do that, I doubt it would be difficult to make the book page work with their extension, if it wouldn't already.

**Reply by eirenicon** [reply]
I know I use that function all the time on GoodReads.  It truly helps me find more available ebooks than anything else. 

I hope use of the plugin  will make Bookwyrm much more attractive to new users, going forward.  I am very happy to help test should we get to that point.

**Reply by mouse-reeve** [reply]
Have you reached out to the developers of the extension? If you know if their code is public, I could make an issue or PR, but I didn't see a link on their website.

**Reply by eirenicon** [reply]
Here is their site... https://www.libraryextension.com/

I have never dealt with them personally. I'm simply a very happy user.

Here's their contact: https://www.libraryextension.com/contact-us

**Reply by dnoesgaard** [reply]
Did anyone ever speak to the Library Extension developers about adding support for Bookwyrm?

It seems their plugin only works on a limited number of URLs so each Bookwyrm instance would need to be added to work.
